src/shared/features/AppletSettings/LiveResponseStreamingSetting/LiveResponseStreamingSetting.test.tsx

```diff
@@ -123,6 +123,6 @@ describe('LiveResponseStreamingSetting', () => {
 
     fireEvent.blur(ipAddressInput);
 
-    expect(getByText('Invalid Port')).toBeInTheDocument();
```

Comment:
Please extract findByText and not import screen:

``const { getByTestId, getByLabelText, findByText } = renderWithAppletFormData(...``

Reply:
I've also corrected this test case in my previous PR, which is merged into develop.